src/js/components/InfiniteScroll/index.d.ts

```diff
@@ -1,10 +1,10 @@
 import * as React from "react";
 
 export interface InfiniteScrollProps {
-  children?: ((...args: any[]) => any);
-  items?: any[];
-  onMore?: ((...args: any[]) => any);
-  renderMarker?: ((...args: any[]) => any);
+  children?: ((...args: [string, number] ) => React.ElementType);
```

Comment:
This failed the test I attached.

Reply:
Commited a change that made everything ok with the file you attached :)
Everythings seens to bem running fine with the types. Many thanks for the help